echopype/echodata/echodata.py

```diff
@@ -478,16 +458,15 @@ def compute_range(
             tvg_correction_factor = TVG_CORRECTION_FACTOR[self.sonar_model]
 
             if waveform_mode == "CW":
-                if (
-                    self.sonar_model == "EK80"
-                    and encode_mode == "power"
-                    and self.beam_power is not None
-                ):
-                    # if both CW and BB exist and beam_power group is not empty
-                    # this means that CW is recorded in power/angle mode
-                    beam = self.beam_power
+                try:
+                    if self.sonar_model == "EK80" and encode_mode == "power":
+                        # if both CW and BB exist and beam_power group is not empty
+                        # this means that CW is recorded in power/angle mode
+                        beam = self["Sonar/Beam_group2"]
+                except GroupNotFoundError:
+                    beam = self["Sonar/Beam_group1"]
                 else:
-                    beam = self.beam
+                    beam = self["Sonar/Beam_group1"]
```

Comment:
Same as comments in `calibrate_ek.py` for an identical block.

Seeing so many of these, I wonder if we should have a util function to do this triage.

Reply:
@leewujung Previously `ed.beam` and such results in `None`. Now we have `GroupNotFoundError` instead. I almost wonder if I should just make it `None` rather than error out?